src/connection_string.ts

```diff
@@ -613,11 +619,13 @@ export const OPTIONS = {
       const compressionList = new Set();
       for (const compVal of values as string[]) {
         for (const c of compVal.split(',')) {
-          if (['none', 'snappy', 'zlib'].includes(String(c))) {
+          if (Object.values(VALID_COMPRESSION_MECHS).includes(String(c))) {
```

Comment:
```suggestion
          if (Object.keys(Compressor).includes(String(c))) {
```

We already have an enum `Compressor` lets use that here

Reply:
Oops, didn't realize there was an existing enum already 😅